Runtime/Scripts/ExampleRest.cs

```diff
@@ -25,7 +25,9 @@
 using System.Net.Http;
 using System;
 using System.Collections.Generic;
+using static DistributedMatchEngine.ServerEdgeEvent.Types;
 
+[RequireComponent(typeof(PersistentConnection))]
```

Comment:
I tried but there is already EdgeEventsConnection with coming from C#, Unity is just a wrapper.

Reply:
"PersistentConnection" should not be used, since it is implementation specific and doesn't really say anything about the function. I suggest just "EdgeEvents" since "EdgeEventsConnection" is already used